modules/launcher/src/main/java/org/mule/module/launcher/domain/DomainMuleContextBuilder.java

```diff
@@ -36,9 +38,10 @@
 
     private final String domainId;
 
-    public DomainMuleContextBuilder(String domainId)
+    public DomainMuleContextBuilder(String domainId, Properties configurationManagementProperties)
```

Comment:
Builders should receive the deployment properties through a setter method and not the constructor.